reg/registry_win.go

```diff
@@ -1,11 +1,12 @@
 //go:build windows
 // +build windows
 
-package winreg
+package reg

```

Comment:
I think `reg` is a bit too simple name, I preferred the old `winreg` it's a bit clearer.

Another thing to think about, as we add more tasks we will also have more of these helper packages that are mostly related to one task. I'm not sure it makes sense to keep them in the root folder. If they're specific to one task it might make more sense to just put them in that package? Or create another subfolder for them.

Reply:
got any ideas for an alternative? `winreg` is currently used by the task and `registry` is used by golang windows. i was struggling for an alternative and didn't really want to use `winreg` twice especially when the packages are functionally pretty different.

see comment above introducing the `tasks/support` directory.

Reply:
thanks. `winregistry` works. i've renamed.